docs/pages/contributing.md

```diff
@@ -80,11 +81,12 @@ pytest -svr A tests
 Running the commands below in your console will show performance data.  You can find the latest results [here]({{site.baseurl}}/performance/).
 
 ```bash
-# install pytest and other dependencies.
+# install pytest and other dependencies
 pip install -r requirements-test.txt
+pip install openpyxl
```

Comment:
This is probably relevant to unit tests, too, not just performance benchmarks.  So, it either needs to be reference in the `pip install` code block above or perhaps added to the `requirements-test.txt` file?  Which is better?

Reply:
`openpyxl ` is no longer needed because we migrated sample data to csv from xlsx. I need to update conftest.py to load data from csv and then remove this dependency.

Reply:
ok, I’ve left us a reminder in #331 and will remove references to it in this PR